R/treeview.R

```diff
@@ -562,21 +481,24 @@ treeview <- function(e0,
     suppressWarnings(.plot_tree(vn, mut_regex = mutations))
   }
 
-  suppressWarnings(
-    .pl_cluster_sina(pldf,
+  suppressWarnings({
+    plot_cluster_sina(
+      pldf,
+      output_dir = output_dir,
```

Comment:
Assuming this just uses the default value of `varx = "logistic_growth_rate"` hence why the argument isn't specified. Same as original code so fine with this :)